website/docs/r/attestation.html.markdown

```diff
@@ -0,0 +1,111 @@
+---
+subcategory: "Attestation"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_attestation"
+description: |-
+  Manages a Attestation Resource.
+---
+
+# azurerm_attestation
+
+Manages a Attestation Resource.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "UK South"
+}
+
+resource "azurerm_attestation" "example" {
+  name                = "example-attestationprovider"
+  resource_group_name = azurerm_resource_group.example.name
+  location            = azurerm_resource_group.example.location
+}
+```
```

Comment:
I think we can remove this example, since the file one should be the most common?

Reply:
Agreed, if needed I can add a sample of this in the example directory later.

Reply:
Fixed